sdk/resourcemanager/Azure.ResourceManager.Core/src/Generated/SubscriptionContainer.cs

```diff
@@ -125,16 +130,6 @@ async Task<Page<Subscription>> NextPageFunc(string nextLink, int? pageSizeHint)
             return PageableHelpers.CreateAsyncEnumerable(FirstPageFunc, NextPageFunc);
         }
 
-        /// <summary>
-        /// Validate the resource identifier is supported in the current container.
-        /// </summary>
-        /// <param name="identifier"> The identifier of the resource. </param>
-        protected override void Validate(ResourceIdentifier identifier)
-        {
-            if (!(identifier is null))
-                throw new ArgumentException("Invalid parent for subscription container", nameof(identifier));
-        }
-
```

Comment:
No need to override `Validate()` after changing the `ValidResourceType` to `TenantOperations.ResourceType` which is the correct type for the parent id of `SubscriptionContainer`.